diagnostic.md

```diff
@@ -6,36 +6,46 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    - The Ember Router tells the parses the requests from the client and calls
+    functions that will display react and render things according to the
+     request.
+     - a route will determine where and how data is send down and actions are
+       sent up.
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember g route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    The question states the route we have, and that we need a 'link-to', but not
+    what we are linking to or what an appropriate link is, or where we are
+    linking from. What am i missing?
+    {{#link-to 'campus.boston'}}Link to Boston Campus{{/link-to}}
     ```
 
 1.  Explain **at least** two differences between the following two route
     definitions.
 
     ```js
     this.route('products', function () {
-      this.route('product', { path: '/:product_id' }); // <= 👀here
+      this.route('product', { path: '/:product_id' }); // <= 👀 here
     });
 
     this.route('product', { path: '/products/:product_id' }); // <= 👀 here
     ```
 
     ```md
-    <!-- your response here -->
+    - the first definition is does not nest individual products under the
+    products route.
+    - the first block of code nests 'this.route' in a callback for
+    another 'this.route'.
```

Comment:
.5/1 

The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.